canonical-web-and-design/ubuntu.com
# Load #get-in-touch form on button click

## Done

- Load #get-in-touch form on button click
- Create new #get-in-touch for openstack on pricing/consulting

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [pricing/consulting](http://0.0.0.0:8001/pricing/consulting)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to [pricing/consulting](http://0.0.0.0:8001/pricing/consulting) and click the "Get in touch" button in the OpenStack section. Compare with [copy doc](https://docs.google.com/document/d/1PG64GkPBA-Y7LR0OWYEIKWE-mD6BDgbgiTgTNisB-20/edit#heading=h.8j6e6xp4ehkf). If you press any other "Get in touch" button on this page, the form should be different.
- Go to any page that has a "Get in touch" and make sure the form is displayed on button click.


## Issue / Card

Fixes #5013
